docs/validators.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Validation functions
+author: Roxane Letourneau
+---
+
+Taquito provides functions that allow us to see if an address, a chain, a key hash, a contract address, a public key or a signature is valid. The `ValidationResult` returned by these functions is an enum which can take the following values:
+  ```
+  0 = NO_PREFIX_MATCHED,
+  1 = INVALID_CHECKSUM,
+  2 = INVALID_LENGTH,
+  3 = VALID
+  ```
+
+### Validate an address
+
```

Comment:
It would be good to group

- validateAddress (Will validate an implicit (tz1/2/3 or a KT1 address))
- validateKeyHash (Will validate only implicit addresses (tz1/2/3))
- validateContractAddress (Will validate only contract address (KT1))

together, and explain why they are different. 

The naming of these methods could be better, but that's a different matter.